internal/oci/verifier.go

```diff
@@ -43,6 +45,10 @@ type options struct {
 	PublicKey  []byte
 	ROpt       []remote.Option
 	Identities []cosign.Identity
+	TrustStore *trustpolicy.Document
```

Comment:
```suggestion
	TrustPolicy *trustpolicy.Document
```

Reply:
Resolved in [0720470](https://github.com/fluxcd/source-controller/pull/1075/commits/0720470ce2b32f8568ab7390e439e51b8bb5a05c).